mysql/datadog_checks/mysql/statements.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import copy
+import logging
+from contextlib import closing
+
+import pymysql
+
+from datadog_checks.base.utils.db.sql import compute_sql_signature
+from datadog_checks.base.utils.db.statement_metrics import StatementMetrics, apply_row_limits
+
+try:
+    import datadog_agent
+except ImportError:
+    from ..stubs import datadog_agent
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Happy to change it if that's the correct pattern to follow, but just out of curiosity, what is the advantage of this? Isn't the whole point of python's logging framework to avoid having to store and pass around a logger object?

Reply:
The main goal here is to keep the same logging context inside an integration, so it's easier to read